src/Sparse/Matrix.jl

```diff
@@ -23,6 +23,37 @@ parent(A::SMat) = SMatSpace(base_ring(A), A.r, A.c)
 
 base_ring(A::SMat{T}) where {T} = A.base_ring::parent_type(T)
 
+base_ring_type(::Type{<:SMat{T}}) where {T} = parent_type(T)
```

Comment:
This is of course not necessary for this PR, but I noticed it was "missing", and thought I should just add it while I am here...